unittests/finality_tests.cpp

```diff
@@ -453,6 +454,7 @@ BOOST_AUTO_TEST_CASE(delayed_strong_weak_lost_vote) { try {
 
    BOOST_REQUIRE(cluster.produce_blocks_and_verify_lib_advancing());
 } FC_LOG_AND_RETHROW() }
+#endif
```

Comment:
I believe in Boost unit tests, when safety file is the default, those simulated weak vote tests should have passed. Should come back to revisit those later.